android/src/main/java/app/notifee/core/utility/PowerManagerUtils.java

```diff
@@ -93,6 +93,26 @@ public static Boolean isBatteryOptimizationEnabled(Context context) {
     return !pm.isIgnoringBatteryOptimizations(context.getPackageName());
   }
 
+  /**
+   * Lights up the screen if it is not already lit
+   *
+   * @param context
+   */
+  public static void lightUpScreenIfNeeded(Context context) {
+    PowerManager pm = (PowerManager) context.getSystemService(Context.POWER_SERVICE);
+    Boolean isScreenOn = pm.isScreenOn();
+
+    if (isScreenOn == false) {
+        PowerManager.WakeLock wl = pm.newWakeLock(
+          PowerManager.FULL_WAKE_LOCK |PowerManager.ACQUIRE_CAUSES_WAKEUP |PowerManager.ON_AFTER_RELEASE, "MyLock"
```

Comment:
not sure if this will make it through java format, may need ` | ` for the boolean or here, we will see

Reply:
hmm, here it worked and passed fine. The actions will take if the format is wrong right?

Reply:
Indeed! All the lint runs have already passed --> :heavy_check_mark: for formatting